repository.md

```diff
@@ -91,7 +91,7 @@ git_repository(
 | <a id="git_repository-commit"></a>commit |  The git commit to check out. Either `commit` or `tag` may be specified.   | String | optional |  `""`  |
 | <a id="git_repository-overlay"></a>overlay |  A set of files to copy into the downloaded repository. The keys in this dictionary are Bazel labels that point to the files to copy. These must be fully qualified labels (i.e., `@repo//pkg:name`) because relative labels are interpreted in the checked out repository, not the repository containing the WORKSPACE file. The values in this dictionary are root-relative paths where the overlay files should be written.<br><br>It's convenient to store the overlay dictionaries for all repositories in a separate .bzl file. See Gazelle's `manifest.bzl`_ for an example.   | <a href="https://bazel.build/rules/lib/dict">Dictionary: Label -> String</a> | optional |  `{}`  |
 | <a id="git_repository-remote"></a>remote |  The remote repository to download.   | String | required |  |
-| <a id="git_repository-repo_mapping"></a>repo_mapping |  In `WORKSPACE` context only: a dictionary from local repository name to global repository name. This allows controls over workspace dependency resolution for dependencies of this repository.<br><br>For example, an entry `"@foo": "@bar"` declares that, for any time this repository depends on `@foo` (such as a dependency on `@foo//some:target`, it should actually resolve that dependency within globally-declared `@bar` (`@bar//some:target`).<br><br>This attribute is _not_ supported in `MODULE.bazel` context (when invoking a repository rule inside a module extension's implementation function).   | <a href="https://bazel.build/rules/lib/dict">Dictionary: String -> String</a> | optional |  |
```

Comment:
This comes from the downgrade to 6.4.0. Let's use 7.0.2 instead.

Reply:
is this from https://github.com/bazelbuild/bazel-gazelle/pull/1731/files#r1503290363 or a different 6.4.0?

Reply:
When i drop this change, I end up with the following failures:

* `//cmd/gazelle:gazelle_test`  -> `could not locate go tool`
* `//internal:bazel_test` -> `error: unknown runfile: ./.bazelrc`

Downgrading to whats run in CI is required to not fail.